Dart/thirdPartySrc/analysisServer/org/dartlang/analysis/server/protocol/Outline.java

```diff
@@ -103,8 +103,8 @@ public static Outline fromJson(Outline parent, JsonObject outlineObject) {
     Element element = Element.fromJson(elementObject);
     int offset = outlineObject.get("offset").getAsInt();
     int length = outlineObject.get("length").getAsInt();
-    int codeOffset = outlineObject.get("codeOffset") == null ? offset : outlineObject.get("codeOffset").getAsInt();
-    int codeLength = outlineObject.get("codeLength") == null ? length : outlineObject.get("codeLength").getAsInt();
+    int codeOffset = outlineObject.get("codeOffset").getAsInt();
+    int codeLength = outlineObject.get("codeLength").getAsInt();
```

Comment:
I'll keep these lines unchanged just for the compatibility with older Dart SDK versions.